container/container_unix.go

```diff
@@ -415,6 +416,19 @@ func (container *Container) TmpfsMounts() []Mount {
 			Data:        data,
 		})
 	}
+	for _, mnt := range container.HostConfig.Mounts {
+		if mnt.Type == mounttypes.TypeTmpfs {
```

Comment:
We should have these added to container.Mountpoints now that we have a proper "Type" field.... it would also be helpful to have `HostConfig.Tmpfs` converted into this format like we are doing (daemon side) for stuff in `Binds` and `Volumes`.